src/com/javabasic/lesson7/CarWash.java

```diff
@@ -0,0 +1,93 @@
+package com.javabasic.lesson7;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.TimeUnit;
+
+public class CarWash {
+
+    static int count;
+    final static BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        System.out.println("How many cars need to wash?");
+        int countOfCarsToWash = isInt();
+
+        System.out.printf("### The washing is started for cars: %s ###\n", countOfCarsToWash);
+        // first way with for
+        for (int i = 1; i <= countOfCarsToWash; i++) {
+            System.out.println("Washing a car: " + i);
+            printStageSoap();
+            TimeUnit.SECONDS.sleep(2);
+            printStageWater();
+            TimeUnit.SECONDS.sleep(2);
+            printStageDry();
+            TimeUnit.SECONDS.sleep(1);
+            System.out.println("The car is washed");
+            if (i < countOfCarsToWash) {
+                System.out.println("* Going to wash the next car *");
+                TimeUnit.SECONDS.sleep(3);
+            }
+        }
+        // second way with do
+        do {
+            count++;
+            System.out.println("Washing a car: " + count);
+            printStageSoap();
+            TimeUnit.SECONDS.sleep(2);
+            printStageWater();
+            TimeUnit.SECONDS.sleep(2);
+            printStageDry();
+            TimeUnit.SECONDS.sleep(1);
+            System.out.println("The car is washed");
+            if (count < countOfCarsToWash) {
+                System.out.println("* Going to wash the next car *");
+                TimeUnit.SECONDS.sleep(3);
+            }
+        } while (count != countOfCarsToWash);
+
+        System.out.println("### The washing is finished. Cars are ready to use ###");
+
+    }
+
+    public static int isInt() throws IOException {
```

Comment:
as this method returns an int value better name is `getInt`, `getNumber`, `getInput`
if the method returns a boolean value we can use an `is` prefix, for example `isPositive` (if the method responds if the number is positive), etc